### PR TITLE
feat: display version of the MPAS CLI

### PR DESCRIPTION
**Pull request by Skarlso**
## Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

- Related Issue # (issue)
- Closes # (issue)
- Fixes # (issue)
> Remove if not applicable

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help
- [ ] Separate ticket for tests # (issue/pr)

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Added e2e-tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help
- [ ] Separate ticket for e2e-tests

## Added to documentation?

- [ ] 📜 README.md
- [ ] 🙅 no documentation needed

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Skarlso** [comment]
Closes https://github.com/open-component-model/MPAS/issues/108

**Comment by Skarlso** [comment]
```
make build-dev
go build -ldflags="-s -w -X main.Version=0.0.0-add-version-62ea523-1698671235" -o ./bin/mpas ./cmd/mpas
➜  MPAS git:(add-version) ./bin/mpas version
0.0.0-add-version-62ea523-1698671235

```

File: cmd/mpas/version.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+)
+
+// NewVersion returns a new cobra.Command to provide version information.
+func NewVersion() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "version",
+		Short: "returns the version of the current binary",
+		Long:  "returns the version of the current binary",
+		Run: func(cmd *cobra.Command, args []string) {
+			fmt.Println(Version)
```
Comment (souleb) [inline cmd/mpas/version.go:20]: we could use our printer.

**Review by souleb — APPROVED** [review]
minor comment but otherwise gtm.

**Comment by Skarlso** [comment]
With the printer:

```
./bin/mpas version
0.0.0-add-version-62ea523-1698673288

```

**Pull request merged**